mountinfo/mounted_linux_test.go

```diff
@@ -314,6 +316,47 @@ func TestMountedBy(t *testing.T) {
 				}
 			}
 
+			// Check the public MountedFast() function as a whole.
+			mounted, sure, err := MountedFast(m)
+			// In case of an error, sure and mounted must be false.
+			if err != nil {
+				// Got an error; is it expected?
+				if !(tc.isNotExist && errors.Is(err, os.ErrNotExist)) {
+					t.Errorf("MountedFast: unexpected error: %v", err)
+				}
+				if sure {
+					t.Error("MountedFast: expected sure to be false on error")
+				}
+				if mounted {
+					t.Error("MountedFast: expected mounted to be false on error")
+				}
+			} else {
+				if openat2Supported {
+					if mounted != exp {
```

Comment:
This test is becoming quite complex; could we remove the local `exp` variable, and instead use `tc.isMount` ? (I had to scroll up to learn what `exp` was)

Reply:
fixed